libs/perun/components/src/lib/members-dynamic-list/members-dynamic-list.component.html

```diff
@@ -40,23 +40,28 @@
           <td *matCellDef="let member" mat-cell>{{member.user | userFullName}}</td>
         </ng-container>
         <ng-container matColumnDef="status">
-          <div *ngIf="showGroupStatuses then thenHeader else elseHeader"></div>
-          <ng-template #thenHeader>
-            <th *matHeaderCellDef mat-header-cell>{{'MEMBERS_LIST.GROUP_STATUS' | translate}}</th>
-          </ng-template>
-          <ng-template #elseHeader>
-            <th *matHeaderCellDef mat-header-cell>{{'MEMBERS_LIST.STATUS' | translate}}</th>
-          </ng-template>
+          <th *matHeaderCellDef mat-header-cell>{{'MEMBERS_LIST.STATUS' | translate}}</th>
           <td *matCellDef="let member" mat-cell>
             <i (click)="changeStatus($event, member)"
-               class="material-icons {{showGroupStatuses ? (member.groupStatus | memberStatusIconColor) : (member.status | memberStatusIconColor)}}"
-               matTooltip="{{member | memberStatusTooltip: showGroupStatuses}}"
+               class="material-icons {{(member.status | memberStatusIconColor)}}"
+               matTooltip="{{member | memberStatusTooltip: false}}"
                matTooltipClass="status-tooltip"
                matTooltipPosition="left">
-              <span *ngIf="!showGroupStatuses">
+              <span>
                 {{member.status | memberStatusIcon}}
               </span>
-              <span *ngIf="showGroupStatuses">
+            </i>
+          </td>
+        </ng-container>
+        <ng-container matColumnDef="groupStatus">
```

Comment:
This component is missing the "INDIRECT" column, which can be used to show the origin of a group member.